javascript/ql/test/query-tests/Security/CWE-079/tst3.js

```diff
@@ -4,6 +4,11 @@ var data = JSON.parse(decodeURIComponent(window.location.search.substr(1)));
 foo.setAttribute("src", data.src); // NOT OK
 foo.setAttribute("HREF", data.p);  // NOT OK
 foo.setAttribute("width", data.w); // OK
+foo.setAttribute("xlink:href", data.p) // NOT OK
+
+foo.setAttributeNS('foobar', 'href', data.p); // NOT OK
```

Comment:
Why "foobar"? Isn't `xlink` from above a very natural example?

Reply:
Yes, `xlink` is a natural example. 
However, I wanted the test to explicitly show that the query matches the `href` attribute with any namespace when using the `setAttributeNS` method. 
(This was the case already before my change in DOM.qll)

Reply:
Then why not have both?

Reply:
I'll do that.